test/unit/selector.js

```diff
@@ -480,18 +1985,34 @@ QUnit.test( "jQuery.uniqueSort", function( assert ) {
 		}
 	};
 
+	// Trigger a dummy query to reset the document to the default one as in 3.x
+	// that matters for the sorting order.
+	jQuery( "#qunit-fixture" );
```

Comment:
I think this shouldn't really matter anymore since we started invoking `setDocument` inside of `contains` that's used in this API. I'll need to check & remove if not needed anymore.

Reply:
Removed.